nubis/terraform/main.tf

```diff
@@ -173,3 +173,14 @@ module "backup" {
   purpose      = "backup"
   role         = "${module.worker.role}"
 }
+
+module "cloudfront" {
+  source                 = "github.com/nubisproject/nubis-terraform//cloudfront?ref=131daeedc94660a39fb8de148706070c6d293cf2"
```

Comment:
Please add a comment explaining why that specific commit is needed ( a link to the github issue works just fine )